base/all-all/sbardef.lmp

```diff
@@ -866,6 +866,21 @@
               "conditions": null,
               "children": null
             }
+          },
```

Comment:
Does this mean that whenever a PWAD provides its own SBARDEF lump, such as Legacy of Rust, we lose the ability to show messages?

Reply:
Right, I forgot about that! Fixed here: https://github.com/fabiangreffrath/woof/pull/1916/commits/ede3bc063a52e3cfec31ddc99f19ba205092bfbf